plugins/modules/nd_interface_flow_rules.py

```diff
@@ -0,0 +1,440 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# Copyright: (c) 2024, Gaspard Micol (@gmicol) <gmicolg@cisco.com>
+# GNU General Public License v3.0+ (see LICENSE or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+
+__metaclass__ = type
+
+ANSIBLE_METADATA = {"metadata_version": "1.1", "status": ["preview"], "supported_by": "community"}
+
+DOCUMENTATION = r"""
+---
+module: nd_interface_flow_rules
+version_added: "0.2.0"
+short_description: Manage Interface Flow Rules
+description:
+- Manage Interface Flow Rules on Cisco Nexus Dashboard Insights (NDI).
+author:
+- Gaspard Micol (@gmicol)
+options:
+  insights_group:
+    description:
+    - The name of the insights group.
+    type: str
+    aliases: [ fab_name, ig_name ]
+  site_name:
+    description:
+    - The name of the ACI Fabric.
+    type: str
+    aliases: [ site ]
+  flow_rule:
+    description:
+    - The name of the Interface Flow Rule.
+    type: str
+    aliases: [ interface_flow_rule, flow_rule_name, name ]
+  flow_rule_type:
+    description:
+    - The type of Interface Flow Rule.
+    - It defines what could be configured in I(nodes).
+    type: str
+    choices: [ port_channel, physical, l3out_sub_interface, l3out_svi ]
+    aliases: [ type ]
+  flow_rule_status:
+    description:
+    - The state of the Interface Flow Rule.
```

Comment:
Would call it status to avoid confusing with our `state` input argument.

```suggestion
    - The status of the Interface Flow Rule.
```

Reply:
yes you're right, it's less confusing.